docs/modules/ROOT/pages/how-tos/synthesize.adoc

```diff
@@ -0,0 +1,256 @@
+export KUBECONFIG=/path/to/synfra.kubeconfig
+export COMMODORE_API_TOKEN=$(kubectl config view -o jsonpath='{.users[?(@.name == "syn-synfra")].user.token}'  --raw)
+export COMMODORE_API_URL="https://api.syn.vshn.net"
+----
++
+Now you can follow the steps in the https://syn.tools/syn/how-tos/compile-catalog.html#_compilation[Project Syn "Compile a catalog" how-to]
+
+. Add required tokens to Vault.
+Check the documentation for the K8s distribution of your cluster for required secrets.
++
+All distributions require tokens for https://k8up.io[K8up] and the https://github.com/projectsyn/component-cluster-backup[cluster-backup] component. Make sure you've got the https://www.vaultproject.io/docs/install[Vault CLI] available locally and run the following commands:
++
+[source,bash]
+----
+export VAULT_ADDR=https://vault-prod.syn.vshn.net
+export TENANT_ID=<tenant id> <1>
+export CLUSTER_ID=<cluster id> <2>
+
+vault login -method=ldap username=<VSHN username> <3>
+
+# Configure global restic password for K8up
+vault kv put -cas=0 clusters/kv/${TENANT_ID}/${CLUSTER_ID}/global-backup password=$(pwgen -s 32 1)
+
+# Configure restic password for cluster backups
+vault kv put -cas=0 clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cluster-backup password=$(pwgen -s 32 1)
+----
+<1> You can find the tenant ID on {portal-lieutenant-tenants}
+<2> The cluster ID which you chose in step 1a
+<3> Your usual VSHN LDAP login, generally `firstname.lastname`
++
+Check tenant-specific documentation for further tokens to configure in Vault.
++
+[NOTE]
+====
+For K8up,  S3 compatible storage is required.
+If the infrastructure does not provide any S3 compatible storage, you need to find an alternative.
+There are several options:
+
+* You can consume S3 storage from a different provider than the one hosting the cluster.
+* You can setup Minio inside the cluster to present an S3-compatible interface backed by a Kubernetes persistent volume.
+There's a https://github.com/projectsyn/component-minio/[Commodore component] for installing Minio.
+====
+
+. Install Steward on the cluster with the install URL displayed in the Portal
++
+CAUTION: Make sure the `kubectl` command in this step is executed against the cluster to synthesize
++
+[source,bash]
+----
+export KUBECONFIG=/path/to/new/cluster.kubeconfig
+kubectl create -f "${INSTALL_URL}"
+----
++
+[NOTE]
+====
+The install URL is only valid once and is only valid for 30 minutes after creating the cluster in Lieutenant.
+You can check the expiration date of the install URL by inspecting the cluster object on the {synfra-cluster}.
+If you don't see a field `.status.bootstrapToken.tokenValid` the token has either been used or has expired.
+
+[source,bash]
+----
+export CLUSTER_ID=<cluster id of cluster to synthesize>
+export KUBECONFIG=/path/to/synfra.kubeconfig
+
+export LIEUTENANT_NS=lieutenant-prod
+
+kubectl -n ${LIEUTENANT_NS} describe cluster ${CLUSTER_ID}
+----
+
+Use the following command to reset the cluster's bootstrap token
```

Comment:
I'd move that to its own how-to.

Reply:
moved to `explanation/bootstrap-token.adoc` for now